### ISSUE TITLE
SOT on LaSOT 

### ISSUE THREAD

**Issue by Flowerfan** [Zhongdao/UniTrack#26]
Hi, Zhongdao,

Thank you for your great work!

I have tested your code on LaSOT using the crw_resnet18_s3 model by modifying the datasets root in utils.py. 
But the AUC is only 23.02 on this dataset. I'm not sure I got the correct results. Have you tested the unitrack on LaSOT or GOT10k? Could you provide the results on LaSOT and GOT10k at your convenience?

**Comment by Zhongdao** [comment]
Hi @Flowerfan,
We indeed have results on more SOT datasets (we did these during rebuttal). You will find these results in the camera-ready version of our supplementary material, and below:
![image](https://user-images.githubusercontent.com/14846754/139828101-fe2fd6e4-174e-4a87-a24c-5744eeeda0a1.png)

Here UniTrack uses an ImageNet pre-trained ResNet-50 as the appearance model.
Your results (23.02 on LaSOT) seem correct. If you want to obtain a better result I suggest slightly tuning hyperparameters here: https://github.com/Zhongdao/UniTrack/blob/44ae779994247fe69ebe297b7e384cc9e3e00195/test/test_sot_cfnet.py#L44

For instance, change this line https://github.com/Zhongdao/UniTrack/blob/44ae779994247fe69ebe297b7e384cc9e3e00195/test/test_sot_cfnet.py#L56
to num_scale = 5 helps.
